source/simulator/assemblers/stokes.cc

```diff
@@ -501,7 +530,7 @@ namespace aspect
 
       Assert(!this->get_parameters().enable_prescribed_dilation
              ||
-             outputs.template get_additional_output_object<MaterialModel::PrescribedPlasticDilation<dim>>()->dilation.size()
+             outputs.template get_additional_output_object<MaterialModel::PrescribedPlasticDilation<dim>>()->dilation_lhs_term.size()
```

Comment:
do you need to also add an assert for the `dilation_rhs_term` here? just in case someone accidentally sets them to different sizes.